pkg/cmd/login/login.go

```diff
@@ -63,6 +65,12 @@ func WithBrowser(ctx context.Context, oktetoURL string) (*okteto.User, error) {
 	authorizationURL := h.AuthorizationURL()
 	fmt.Println("Authentication will continue in your default browser")
 	if err := open.Start(authorizationURL); err != nil {
+		if strings.Contains(err.Error(), "executable file not found in $PATH") {
+			return nil, errors.UserError{
+				E:    fmt.Errorf("No browser could be found"),
+				Hint: "Check '--token' flag to run this command in server mode. More information can be found here: https://okteto.com/docs/reference/cli#login",
```

Comment:
```suggestion
				Hint: "Use the '--token' flag to run this command in server mode. More information can be found here: https://okteto.com/docs/reference/cli#login",
```

Reply:
I'd use 'headless' instead of 'server' mode.